src/main/java/dk/dbc/updateservice/actions/UpdateRequestAction.java

```diff
@@ -153,4 +153,22 @@ String sanityCheckRecord() {
             return message;
         });
     }
+
+    private String getIndicators(DataField field) {
+        final StringBuilder sb = new StringBuilder();
+        if (field.getInd1() == null) {
```

Comment:
Kan man ikke risikere at den er != null men tom ? samme for ind2

Reply:
ind1 er en char og ikke en string, og en char kan ikke være tom. Enten er den null ellers har den en værdi.

Jeg lærte en del om chars da jeg lavede den funktion :)